terraform/modules/cloudwatch/policies.tf

```
@@ -7,7 +7,7 @@ data "aws_iam_policy_document" "elasticsearch-log-publishing-policy" {
       "logs:PutLogEvents",
     ]
 
-    resources = ["arn:aws:logs:*"]
+    resources = ["arn:aws-us-gov:logs:*"]
```

```suggestion
    resources = ["arn:${var.aws_partition}:logs:*"]
```

I would add `aws_partition` as a variable to this module. Then update this block in `stacks/main/stack.tf` like so:

```
module "cloudwatch" {
  source = "../../modules/cloudwatch"

  stack_description                   = var.stack_description
  cg_platform_notifications_arn       = module.sns.cg_platform_notifications_arn
  cg_platform_slack_notifications_arn = module.sns.cg_platform_slack_notifications_arn
  load_balancer_dns                   = module.cf.lb_arn_suffix
  aws_partition =  data.aws_partition.current.partition
}
```

It keeps us from hardcoding a value and will ensure it is deployed with the correct value

---

Good call!